lib/maze/client/appium/base_client.rb

```diff
@@ -22,7 +22,10 @@ def start_session
                                when 'android'
                                  Maze.driver.session_capabilities['appPackage']
                                when 'ios'
-                                 Maze.driver.session_capabilities['CFBundleIdentifier'] # Present on BS and locally
+                                 unless app_id = Maze.driver.session_capabilities['CFBundleIdentifier']
+                                    app_id = Maze.driver.session_capabilities['bundleID']
+                                 end
+                                 app_id
```

Comment:
Wow, nice detective work on this.  I'm guessing we can't just use `bundleID` every time?  I didn't even remember us having this logic in here to start with to know.